src/rollbar.php

```diff
@@ -122,6 +122,7 @@ class RollbarNotifier {
         'scrub_fields', 'shift_function', 'timeout', 'report_suppressed', 'use_error_reporting', 'proxy');
 
     // cached values for request/server/person data
+    private $_context = null;
```

Comment:
Sure, was actually planning to do it alrdy, check the pr description :)
I'll be happy to make any changes you guys have to proppose or implement
other ideas.

On Wed, Apr 13, 2016, 23:04 Brian Rue <notifications@github.com> wrote:

> In src/rollbar.php
> <https://github.com/rollbar/rollbar-php/pull/80#discussion_r59625282>:
>
> > @@ -122,6 +122,7 @@ class RollbarNotifier {
> >          'scrub_fields', 'shift_function', 'timeout', 'report_suppressed', 'use_error_reporting', 'proxy');
> >
> >      // cached values for request/server/person data
> > +    private $_context = null;
>
> Sorta minor, but can you call this variable something besides _context?
> One of the fields that the Rollbar API understands is also "context" and I
> worry that this could be confused with that.
>
> Just as some ideas, maybe _execution_context or _php_context ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/rollbar/rollbar-php/pull/80/files/deb80620fecb2fa78ce2d0c260a5ace744f8f6d5#r59625282>
>